context_test.cc

```diff
@@ -0,0 +1,20 @@
+#include "include/proxy-wasm/context.h"
```

Comment:
nit: perhaps move this into a test directory to avoid polluting the root with other files that will probably come later?

Reply:
OK, but I want to leave the single BUILD file in the top level directory so that it can be overwritten by http_archive trivially.